skietbaan_fe/src/components/Groups.js

```diff
@@ -1,117 +1,149 @@
+     window.location = "/GroupDone";
   }
   toggleHighlight = event => {
-     if (this.state.posts[(event)].highlighted  === true) {
-       this.state.posts[(event)].highlighted=false;
-       {this.setState({count:this.state.count-1})}
-     }
-     else {
-       this.state.posts[(event)].highlighted=true;
-       {this.setState({count:this.state.count+1})}
-     }
+    console.log(event);
+    if (this.state.posts[event].highlighted === true) {
+      this.state.posts[event].highlighted = false;
+      {
+        this.setState({ count: this.state.count - 1 });
+      }
+    } else {
+      this.state.posts[event].highlighted = true;
+      {
+        this.setState({ count: this.state.count + 1 });
+      }
+    }
+  };
+  onBack() {
+    window.location = "/AddGroup";
   }
   render() {
-    const postItems = (   
-      <Table striped hover condensed>
-        <tbody >
-          {this.state.posts.map((post,index) => (
-            <tr key={post.id.toString()}  onClick={()=>this.toggleHighlight(index)} value={post.id} onChange={()=>this.onChange(post.id)}>
-              <td ><h5>{post.username} {index}</h5>
-                <h5>{post.email}</h5></td>
-            </tr>
+  //  console.log(this.props.groupNames.groupName)
+    const postitems = (
+      <div className="check">
+        <ul class="list-group">
+
+          {this.state.posts.filter(
+            (post) => {
+              return (!this.state.filterText || post.username.toLowerCase().startsWith(this.state.filterText.toLowerCase()) || post.email.toLowerCase().startsWith(this.state.filterText.toLowerCase()))
+            }
+          ).map((post, index) => (
+            <li class="list-group-item list-group-item-light" key={post.id}>
+              <input
+                type="checkbox"
+                className="boxs"
+                onClick={() => this.toggleHighlight(index)}
+              />
+              <label className="blabe">
+                {" "}
+                {post.username} <br />
+                {post.email}
+              </label>
+            </li>
           ))}
-        </tbody>
-      </Table>
+        </ul>
+      </div>
     );
     return (
-      <main>
-        <div main className="topnav" data-simplebar data-simplebar-auto-hide="false">
-          <label className="create">Create Groups</label>
-         
-          <label className="label" onChange={this.handleOnChange}>{this.state.count}</label>
-          <br />
-          <a className="search" href="/Groups"><span class="glyphicon glyphicon-chevron-left"></span></a>
-          <input className="texts" type="text" placeholder="Search.." />
+      <main className="TheMain" >
+        <div className="TheNavBar">
+          <a href="#" class="fa fa-angle-left" onClick={this.onBack} />
+          <div className="center_label">
+            <b>Retro Rabbit</b>
+          </div>
+        </div>
+        <div className="BNavBar">
+          <input
+            className="theText"
```

Comment:
Hey please keep consistent with your naming scheme `CapitalCase`.

Reply:
made the chenges